app/src/components/Home/CoverPage.jsx

```diff
@@ -40,106 +42,98 @@ class CoverPage extends Component {
 
   scrollPage() {
     const el = document.getElementById('steps');
-    if (!el) return;
+    if (!el) {
+      return;
+    }
     scrollTo(el);
   }
 
+  renderQuoteSlide(coverPageEntry) {
+    return (
+      <div key={coverPageEntry.quote} className={CoverPageStyle['leo-slider']}>
+        <div className={CoverPageStyle['contain-quote-text']}>
+          <blockquote>
+            {coverPageEntry.quote}
+          </blockquote>
+          <p className={CoverPageStyle['author-quote']}>– {coverPageEntry.author}</p>
+          <p>{coverPageEntry.subtitle}</p>
+        </div>
+      </div>
+    );
+  }
+
+  renderStandardSlide(coverPageEntry) {
+    return (
+      <div key={coverPageEntry.title}>
+        <div className={CoverPageStyle['contain-text-cover']}>
+          <h1>
+            {coverPageEntry.title}
+          </h1>
+          <p>
+            {coverPageEntry.subtitle}
+          </p>
+        </div>
+      </div>
+    );
+  }
+
   render() {
+    const loadedEntries = (this.props.coverPageEntries && this.props.coverPageEntries.length > 0);
+
     const settings = {
-      dots: true,
+      dots: loadedEntries,
       arrows: false,
       dotsClass: CoverPageStyle['dots-cover'],
       infinite: true,
       draggable: false,
-      afterChange: (currentSlider) => { this.onSliderChange(currentSlider); },
-      autoplay: this.state.autoPlaySlider,
+      afterChange: (currentSlider) => {
+        this.onSliderChange(currentSlider);
+      },
+      autoplay: loadedEntries && this.state.autoPlaySlider,
       autoplaySpeed: this.state.speedPlaySlider
     };
 
-    const sliderBackgrounds = [
-      sliderBackground1,
-      sliderBackgroundLDF,
-      sliderBackground3,
-      sliderBackground4,
-      sliderBackground5
-    ];
-    const sliderBackground = sliderBackgrounds[this.state.currentSlider];
-
-    const sliderAttributions = [
-      '© OCEANA / Juan Cuetos',
-      null,
-      '© Steve De Neef ',
-      '© OCEANA / Eduardo Sorensen',
-      'Hoatzinexp/iStock/Thinkstock'
-    ];
-    const sliderAttribution = sliderAttributions[this.state.currentSlider];
+    let coverEntriesContent = (
+      <div>
+        <div key={-1}>
+          <div className={CoverPageStyle['contain-text-cover']}>
+            <h1 className={CoverPageStyle.stub}>&nbsp;<br />&nbsp;</h1>
+            <p>&nbsp;<br />&nbsp;</p>
+          </div>
+        </div>
+      </div>
+    );
```

Comment:
The idea here is that, if the json is hosted somewhere else, it might take a bit to load. So I tried to make something emtpy-ish but that, at the same time, doesn't use a loading indicator or something that causes a big change when we load the actual content of the slider